### PR TITLE
graphics/drawio: + Diagramming and whiteboarding desktop app

### PR DESCRIPTION
**Pull request by Zoid-dr**
Co-authored-by: Alexey Donskov <voxnod@gmail.com>

Sponsored by:   Serenity Cybersecurity, LLC

### CODE REVIEW

File: graphics/drawio/Makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+MAINTAINER=	636808@mail.ru
+COMMENT=	Diagramming and whiteboarding desktop app
+WWW=		https://drawio.com/
+
+LICENSE=	APACHE20
+
+USES=		desktop-file-utils electronfix:25 gettext-runtime gl gnome \
+			iconv jpeg xorg
+
+NO_WRKSUBDIR=	yes
```
Comment (arrowd) [inline graphics/drawio/Makefile]: Move this near `SUB_FILES`.

File: graphics/drawio/Makefile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+USE_GL=		gbm gl glesv2
+USE_GNOME=	cairo gdkpixbuf2 glib20 gtk30 libxml2 libxslt
+USE_XORG=	x11 xcb xcomposite xcursor xdamage xext xfixes xi xrandr \
+			xrender xscrnsaver xtst
+
+ELECTRONFIX_MAIN_EXECUTABLE=	drawio
+SUB_FILES=	drawio
+NO_BUILD=	yes
+STRIP=
```
Comment (arrowd) [inline graphics/drawio/Makefile:26]: This is already done by `USES=electronfix`, remove.

File: graphics/drawio/Makefile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+do-install:
+	cd ${WRKSRC}/opt/${PORTNAME} && ${COPYTREE_SHARE} \* ${STAGEDIR}${DATADIR}
```
Comment (arrowd) [inline graphics/drawio/Makefile]: The more canonical way is to `cd ${WRKSRC}/opt/` and then `${COPYTREE_SHARE} drawio ...`

File: graphics/drawio/Makefile [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+	cd ${WRKSRC}/usr && ${COPYTREE_SHARE} \* ${STAGEDIR}${PREFIX}
```
Comment (arrowd) [inline graphics/drawio/Makefile]: What does this do specifically? If you want to copy `share/` then `cd ${WKRSRC}/usr && ${COPYTREE_SHARE} share ...`

**Review by arrowd — CHANGES_REQUESTED** [review]

File: graphics/drawio/Makefile (hunk 1 above)
Comment (Zoid-dr) [reply graphics/drawio/Makefile]: Don't work in USES

File: graphics/drawio/Makefile (hunk 3 above)
Comment (Zoid-dr) [reply graphics/drawio/Makefile]: Don't work in USES

File: graphics/drawio/Makefile (hunk 2 above)
Comment (Zoid-dr) [reply graphics/drawio/Makefile:26]: Don't work in USES

**Pull request merged**